test/integration/import_int_test.go

```diff
@@ -86,6 +112,14 @@ func (suite *ImportIntegrationTestSuite) TestImport() {
 		cp.Expect("already exists")
 		cp.ExpectNotExitCode(0)
 	})
+
+	suite.Run("complex requirements.txt", func() {
+		ts.SetT(suite.T())
+		ts.PrepareFile(reqsFilePath, complexReqsData)
+
+		cp := ts.Spawn("import", "requirements.txt")
+		cp.ExpectExitCode(0)
```

Comment:
Can you add a `state packages` and ensure some of the requirements are added? Additionally, we should add an expect for the `Auto` version with a link to a follow-up story for the actual version numbers once we have that functionality.